R/reportWaterUsage.R

```diff
@@ -17,19 +17,19 @@
 
 reportWaterUsage<-function(gdx, detail=TRUE) {
   
-  out <- water_usage(gdx,level="regglo",users=NULL,sum=FALSE,digits=3)[,,"agriculture"]
-  getNames(out) <- "Resources|Water|Withdrawal|Agriculture (km3/yr)"
-   
+  x <- water_usage(gdx, level="regglo",users=NULL, digits=3)[,,"agriculture"]
```

Comment:
I'd keep `sum=FALSE` even though it's a default parameter. Just for the clarity of the code.

Reply:
Updated